Automattic/jetpack
# CRM: Fields deleted if not included in CSV

## Proposed changes:

When importing contacts from a CSV, all columns that are not included in it get deleted in existing contacts. E.g. Including only the columns Email and City would make all other fields (name, last name..,) for existing contacts to be deleted.

This PR fixes this bug.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/zero-bs-crm/issues/2839

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create two contacts using the following e-mails: `do.not.delete@dev.null` - `do.not.delete.2@dev.null`
* Add some information to these contacts (e.g. Name, Last Name, City)
* Import the Sample CSV (see below) `wp-admin/admin.php?page=zerobscrm-csvimporterlite-app`
* Check those two contacts 

In `trunk` contact's information gets erased (e.g. Name, Last Name, City).
In `crm/fix/fields_deleted_if_not_included_in_csv` the information gets preserved, and only the new information gets overwritten.

Sample CSV:
```
Email,Mobile Phone
do.not.delete@dev.null,111111222222
do.not.delete.2@dev.null,333333444444
```
